openseeds/payments/invoicing.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
This page should probably go in the main `policies-procedures-and-docs` folder, not in `openseeds`

Reply:
Done. Thank you!